terraform/environment/ecs_front.tf

```diff
@@ -59,8 +60,8 @@ resource "aws_ecs_task_definition" "front" {
   family                   = "${local.environment}-front"
   requires_compatibilities = ["FARGATE"]
   network_mode             = "awsvpc"
-  cpu                      = 2048
-  memory                   = 4096
+  cpu                      = 256
```

Comment:
were these values too high previously? - just interested....

Reply:
Good question.

I did some analysis on our usage on production and redefined the allocation for all of the services to get the same level of service out of smaller tasks (get rid of under-utilisation).

The utilisation min, max and avg were taken from Cloudwatch across a timeframe of all available data (since we migrated to ECS to now).

![image](https://user-images.githubusercontent.com/6545556/80687201-daa83780-8ac1-11ea-9f7b-cc5c09518950.png)

To stay on the safe side, while I've reduced the CPU and memory, I've kept the minimum task count the same for now.

The result should be no change in performance of the service, auto-scaling to deal with peaks, cheaper ECS services and the scope to scale down a bit more when in low demand.

Reply:
There's potential to do further performance tweaks around how NGINX and PHP interact, but this is really good in terms of making our default scaling appropriate to the needs of the app. Great work!